### ISSUE TITLE
wLUNA stuck in v1

### ISSUE THREAD

**Issue by thesighguy** [solana-labs/oyster#352]
I used wormhole to successfully bridge Luna from ETH chain to SOL chain. When I tried to pull it back days later, it got stuck. The transaction seemed to be successful on SOL (I pulled up the tx), but nothing showed up on ETH side. I never got a prompt to verify the transaction in MM. And yes, I have enough ETH for gas. Please help.

Also, I checked for weird bridge tokens in my ETH wallet. No new tokens, so this isn't a w(wormhole)wLuna problem. It is definitely a "token didn't show up at all" problem.

I also tried waiting on v1 for the "recent transactions" data to show up so I could hit the retry button. There is no data, and the retry button never showed up. I tried the recovery tx input on v2. I got an error saying it could not recognize the tx number I put in.

https://solscan.io/tx/5zDv7KphnxMSeEuz8VdgHuwuNMboez9n8f1zBtjoZR849gNw19MgXcfaAqG1BYGUveLLJ3YrUBUMjsqNZXtZfS6L

**Comment by leoluk** [comment]
Duplicate of https://github.com/certusone/wormhole/discussions/460
